components/Button/index.jsx

```diff
@@ -0,0 +1,133 @@
+import React, {Component, PropTypes} from 'react';
+import {Link} from 'react-router';
+import classNames from 'classnames';
+
+import styles from './index.scss';
+
+import {lighten, darken} from '../../utils/colorChanger';
```

Comment:
This should work like this 'utils/colorChanger';

** I found this out in my autocomplete since I had to use some utils stuff too. We just need to update the docs webpack.base.js to use the right path. See here: https://github.com/kununu/nukleus/pull/46/commits/05227dcf8d9f7ec3a61a2f68d5d22f9c97bde745#diff-2946248d5f33207c803f753e9efbfd0fR24